src/API/Site/Controllers/MerchantCenter/ContactInformationController.php

```diff
@@ -228,7 +234,8 @@ protected function get_contact_information_response( ?AccountBusinessInformation
 		if ( $contact_information instanceof AccountBusinessInformation ) {
 			if ( ! empty( $contact_information->getPhoneNumber() ) ) {
 				try {
-					$phone_number = PhoneNumber::cast( $contact_information->getPhoneNumber() )->get();
+					$phone_number              = PhoneNumber::cast( $contact_information->getPhoneNumber() )->get();
+					$phone_verification_status = strtolower( $contact_information->getPhoneVerificationStatus() );
```

Comment:
💅 Maybe we can have 'VERIFIED' as well in the frontend and get rid of this `strtolower` function

Reply:
Since the status fields returned by the other GLA APIs are almost always lowercase, the idea here was to keep them consistent.